code/game/objects/effects/aliens.dm

```diff
@@ -479,9 +479,18 @@
 			T.dump_contents()
 			qdel(target)
 
+		if(istype(target, /turf/simulated/mineral))
+			var/turf/simulated/mineral/M = target
+			M.ChangeTurf(M.baseturf)
+
+		if(istype(target, /turf/simulated/floor))
+			var/turf/simulated/floor/F = target
+			F.ChangeTurf(F.baseturf)
+

```

Comment:
I would do just simulated, but I fear that would break things. Thus I check for the mineral turfs and floor turfs seperately.